lib/bandit/http1/adapter.ex

```diff
@@ -102,8 +102,8 @@ defmodule Bandit.HTTP1.Adapter do
     end
   end
 
-    # If we do not have a connection header, then keep alive iff we're running on HTTP/1.1
-    defp should_keepalive?(version, nil), do: version == :"HTTP/1.1"
+  # If we do not have a connection header, then keep alive if we're running on HTTP/1.1
```

Comment:
That wasn't actually a typo - iff == if and only if. No matter, the intent of the comment still stands.

Reply:
Ah okay. Didn't knew that. 
Just reverted it back to "iff".